netbox_librenms_plugin/tests/test_coverage_devices.py

```
@@ -0,0 +1,710 @@
+    view.request.path = "/dcim/devices/1/librenms-sync/"
+    view._librenms_api = MagicMock()
+    view._librenms_api.server_key = "default"
+    return view
+
+
+class TestDeviceLibreNMSSyncViewContextMethods:
+    """Tests for DeviceLibreNMSSyncView context delegation (lines 47-73)."""
+
+    def test_get_interface_context_delegates_to_interface_view(self):
+        """get_interface_context() creates DeviceInterfaceTableView and calls get_context_data."""
+        from netbox_librenms_plugin.views.object_sync.devices import DeviceLibreNMSSyncView
```

Flake8 (7.3.0)

[error] 'netbox_librenms_plugin.views.object_sync.devices.DeviceLibreNMSSyncView' imported but unused

(F401)